security/medusa/l3/comm.c

```diff
@@ -4,12 +4,72 @@
 #include <linux/medusa/l3/arch.h>
 #include <linux/medusa/l3/registry.h>
 #include <linux/medusa/l3/server.h>
+#include <linux/medusa/l4/interface.h>
+
 #include "l3_internals.h"
 
+medusa_answer_t development_converter(authserver_answer_t code)
+{
+	medusa_answer_t retval;
+	switch(code) {
+		case AUTHS_ERR:
+			retval = MED_ERR;
+			break;
+		case AUTHS_ALLOW:
+		case AUTHS_DBG_ALLOW:
+		case AUTHS_FORCE_ALLOW:
+		case AUTHS_NOT_REACHED:
+			retval = MED_OK;
+			break;
+		case AUTHS_DENY:
+		case AUTHS_SKIP:
+			retval = MED_NO;
+			break;
+		default:
+			retval = MED_OK;
+	}
+	return retval;
+}
+
+medusa_answer_t production_converter(authserver_answer_t code)
```

Comment:
add comment to this function

Reply:
Missed intentionally, I guess we want to drop support for development codes in production

Reply:
Yes, but this is place for useful comment. Because from view of code it's not clear. If there is set of values namely listed, then in conversion functions must be all of them listed, too.

Reply:
Alright, then from this point of view, I would prefer to list it also in the production. If I will add it as a comment, the comments will get get as soon outdated as when the first person will modify this function.